frontend/src/pages/CompareV2.tsx

```diff
@@ -266,13 +271,28 @@ function CompareV2(props: PageProps) {
   useEffect(() => {
     if (runs && mlmdPackages && artifactTypes) {
       const runArtifacts: RunArtifact[] = getRunArtifacts(runs, mlmdPackages);
-      setConfusionMatrixArtifacts(
+      setConfusionMatrixRunArtifacts(
         filterRunArtifactsByType(runArtifacts, artifactTypes, MetricsType.CONFUSION_MATRIX),
       );
-      setHtmlArtifacts(filterRunArtifactsByType(runArtifacts, artifactTypes, MetricsType.HTML));
-      setMarkdownArtifacts(
+      setHtmlRunArtifacts(filterRunArtifactsByType(runArtifacts, artifactTypes, MetricsType.HTML));
+      setMarkdownRunArtifacts(
         filterRunArtifactsByType(runArtifacts, artifactTypes, MetricsType.MARKDOWN),
       );
+
+      const rocCurveRunArtifacts: RunArtifact[] = filterRunArtifactsByType(
+        runArtifacts,
+        artifactTypes,
+        MetricsType.ROC_CURVE,
+      );
+      const rocCurveArtifacts: Artifact[] = flatMapDeep(
+        rocCurveRunArtifacts.map(rocCurveArtifact =>
+          rocCurveArtifact.executionArtifacts.map(executionArtifact =>
+            executionArtifact.linkedArtifacts.map(linkedArtifact => linkedArtifact.artifact),
+          ),
+        ),
+      );
+      setRocCurveArtifacts(rocCurveArtifacts);
+      setSelectedRocCurveArtifacts(rocCurveArtifacts.slice(0, 3));
```

Comment:
This logic will change when implementing the checkbox / filtering mechanism as I will need to save the `RunArtifact[]` objects in a state variable, but for now, I am just fetching all of the base level `Artifact`s and displaying them on the ROC curve plot.